src/screens/Congratulations/styles.js

```diff
@@ -1,44 +1,62 @@
-import {StyleSheet} from 'react-native';
+import { StyleSheet } from 'react-native';
 
 import {colors, fonts, metrics} from '../../styles';
 
 const styles = StyleSheet.create({
-  container: {
-    flex: 1,
-    backgroundColor: 'rgba(0, 0, 0, 0.8)',
-    justifyContent: 'space-around',
-  },
-  content: {
-    flex: 1,
-    width: metrics.screenWidth,
-    height: metrics.screenHeight,
-    marginTop: 50,
-  },
-  textTop: {
-    flex: 4,
-    textAlign: 'center',
-    color: colors.textColorSecondary,
-    fontSize: metrics.screenWidth * 0.14,
-    fontFamily: 'Poppins-Bold',
+  container:{
+    backgroundColor:colors.colorPrimary,
+    width:metrics.screenWidth,
+    height:metrics.screenHeight,
```

Comment:
```suggestion
    flex: 1,
    padding: metrics.basePadding,
```

Usar flex é a melhor opção, pois vai adaptar o container as diferentes tamanhos de tela. Não use height nem width nos containers das telas, a não ser em casos super específicos.